py-polars/polars/expr/expr.py

```diff
@@ -475,11 +475,12 @@ def any(self, *, ignore_nulls: bool = True) -> Expr:
         Parameters
         ----------
         ignore_nulls
-            Ignore null values (default).
 
-            If set to `False`, `Kleene logic`_ is used to deal with nulls:
-            if the column contains any null values and no `True` values,
-            the output is null.
+            * If set to `True` (default), null values are ignored. If there
+              are no non-null values, the output is `False`.
```

Comment:
I would personally find that sentence slightly easier to understand without the double negation, even if that means a longer sentence. For example:

> If there are only null values or no values at all, the output is `False`.

The same comment applies to most of the other changes in this commit.